listenbrainz/tests/integration/test_playlist_api.py

```diff
@@ -473,6 +586,86 @@ def test_playlist_private_access(self):
+            content_type="application/json"
+        )
+        self.assert200(response)
+        public_playlist_mbid = response.json["playlist_mbid"]
+
+        response = self.client.get(
+            url_for("api_v1.get_playlists_for_user", playlist_user_name=self.user4["musicbrainz_id"]),
+            headers={"Authorization": "Token {}".format(self.user["auth_token"])},
+        )
+        self.assert200(response)
+        self.assertEqual(response.json["playlist_count"], 1)
+
+        response = self.client.get(
+            url_for("api_v1.get_playlists_for_user", playlist_user_name=self.user4["musicbrainz_id"]),
+            headers={"Authorization": "Token {}".format(self.user4["auth_token"])},
+        )
+        self.assert200(response)
+        self.assertEqual(response.json["playlist_count"], 2)
+        self.assertEqual(response.json["playlists"][0]["playlist"]["extension"] \
+                         ["https://musicbrainz.org/doc/jspf#playlist"]["creator"], self.user4["musicbrainz_id"])
+        self.assertEqual(response.json["playlists"][0]["playlist"]["identifier"], PLAYLIST_URI_PREFIX + public_playlist_mbid)
+        self.assertEqual(response.json["playlists"][0]["playlist"]["title"], "1980s flashback jams")
+        self.assertEqual(response.json["playlists"][0]["playlist"]["annotation"], "your lame 80s music")
+        self.assertEqual(response.json["playlists"][0]["playlist"]["extension"] \
+                         ["https://musicbrainz.org/doc/jspf#playlist"]["public"], True)
+        try:
+            dateutil.parser.isoparse(response.json["playlists"][0]["playlist"]["date"])
+        except ValueError:
+            assert False
+        self.assertEqual(response.json["playlists"][1]["playlist"]["extension"] \
+                         ["https://musicbrainz.org/doc/jspf#playlist"]["creator"], self.user4["musicbrainz_id"])
+        self.assertEqual(response.json["playlists"][1]["playlist"]["identifier"], PLAYLIST_URI_PREFIX + private_playlist_mbid)
+        self.assertEqual(response.json["playlists"][1]["playlist"]["title"], "1980s flashback jams")
+        self.assertEqual(response.json["playlists"][1]["playlist"]["annotation"], "your lame 80s music")
+        self.assertEqual(response.json["playlists"][1]["playlist"]["extension"] \
+                         ["https://musicbrainz.org/doc/jspf#playlist"]["public"], False)
+        try:
+            dateutil.parser.isoparse(response.json["playlists"][1]["playlist"]["date"])
+        except ValueError:
+            assert False
+
+        # Test count and offset parameters
+        response = self.client.get(
+            url_for("api_v1.get_playlists_for_user", playlist_user_name=self.user4["musicbrainz_id"], count=1),
+            headers={"Authorization": "Token {}".format(self.user4["auth_token"])},
+        )
+        self.assert200(response)
+        self.assertEqual(len(response.json["playlists"]), 1)
+        self.assertEqual(int(response.json["count"]), 1)
```

Comment:
In this case, 'count' is the parameter, and 'playlist_count' should still be 2, right?

Add count/offset checks to previous test, and add playlist_count check here.

Remove int() cast, would be good to check that these items are always integers, no reason for them to be strings